elements/lisk-validator/src/validation.ts

```diff
@@ -76,7 +77,8 @@ export const isEncryptedPassphrase = (data: string): boolean => {
 	return encryptedPassphraseRegExp.test(data);
 };
 
-export const isSemVer = (version: string): boolean => !!isValidVersion(version);
+export const isSemVer = (version: string): boolean =>
+	!!isValidVersion(coerce(version));
```

Comment:
I think it's not what we should be doing. The `coerce` tries to translate any version string. e.g. If one specifies `v2` it will translate it to `2.0.0` Then this will be validated. But we should not allow people to define application version as `v2`, which is not a valid semantic version. Just use `semver.valid` instead.